src/constants.ts

```diff
@@ -61,6 +61,8 @@ export const acrDomain = 'azurecr.io';
 export const dockerHubDomain = 'docker.io';
 export const dockerHubRegistry = 'index.docker.io';
 
+export const buildImageInAzure = 'buildImageInAzure';
```

Comment:
Nit:  Is this constant used anywhere else outside of `BuildFromProjectListStep`?

If not, I would just declare this constant at the top of the file where it's used and remove it from here.  If we end up needing this in more than one file we can always move it back